components/atom/button/src/index.scss

```diff
@@ -234,17 +231,25 @@ $icon-stroke-color: currentColor !default;
 
   &-centerIcon {
     position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
   }
 
   &#{$base-class}--link:not(#{$base-class}--solid):not(#{$base-class}--outline):not(#{$base-class}--flat) {
+    @include link();
     background: transparent;
-
     border: none;
-    @include link();
-    padding: 0;
+    height: auto;
     justify-content: center;
+    padding: 0;
     text-align: center;
     text-decoration: none;
+
+    #{$base-class}-inner {
+      display: inline;
```

Comment:
ellipsis should work without this modification

Reply:
Really? How? Exactly this is what I found why isn't working before

Reply:
I personally think this is too much specific and we should not increase that danger button for that purposes. Need time to think about it

Reply:
I've changed to this `display: inline` will be changed only when the ellipsis prop is set to true

Reply:
I don't know if it's because there is a different structure but it should work
<img width="205" alt="Captura de pantalla 2020-12-22 a las 15 47 21" src="https://user-images.githubusercontent.com/8382981/102900880-35393e00-446d-11eb-8d6a-fa07b10a6742.png">

Reply:
Tell me how you did it, I'm still trying to figure it out to do that with the link design atom button